nflows/transforms/coupling.py

```diff
@@ -213,16 +213,26 @@ class AffineCouplingTransform(CouplingTransform):
 
     Reference:
     > L. Dinh et al., Density estimation using Real NVP, ICLR 2017.
+
+    The scale is a neural network output transformed by the supplied `scale_activation`
+    then clamped to the interval `scale_min` to `scale_max`.
+    For backwards compatibility, the default `scale_activation` is sigmoid, producing a scale between 0 and 1.
+    In many applications this should be changed -- e.g. to softplus or exponential -- to permit scales above 1.
     """
 
+    def __init__(self, mask, transform_net_create_fn, unconditional_transform=None, scale_activation=torch.sigmoid, scale_min=1e-3, scale_max=3.):
+        self.scale_activation = scale_activation
+        self.scale_min = scale_min
+        self.scale_max = scale_max
+        super().__init__(mask, transform_net_create_fn, unconditional_transform)
+
     def _transform_dim_multiplier(self):
         return 2
 
     def _scale_and_shift(self, transform_params):
         unconstrained_scale = transform_params[:, self.num_transform_features:, ...]
         shift = transform_params[:, : self.num_transform_features, ...]
-        # scale = (F.softplus(unconstrained_scale) + 1e-3).clamp(0, 3)
-        scale = torch.sigmoid(unconstrained_scale + 2) + 1e-3
+        scale = self.scale_activation(unconstrained_scale).clamp(self.scale_min, self.scale_max)
```

Comment:
This will change the default operation, which I think we should try to avoid: it's not great if your model checkpoint stops working after updating the package. Suggestion:  have a `scale_activation` set to, e.g. `DEFAULT_SCALE_ACTIVATION`, which would be a function implementing `torch.sigmoid(unconstrained_scale + 2) + 1e-3`.

`scale_min` and `scale_max` also don't seem super useful: the clamping operation could be done inside `scale_activation` if required, right?

Reply:
Thanks - see the new commits + comment below for a reply to this